src/components/schemas/category.yaml

```diff
@@ -93,3 +102,11 @@ category_in:
       description: Represents the name of the system_category.
     code:
       $ref: ./fields.yaml#/code
+    project_ids:
+      type: array
+      nullable: True
+      items:
+        $ref: './fields.yaml#/id_integer'
+      example: [2222, 1221]
+      description: |
+        List of project ids that this category is allowed to access. If null, means all the projects can access.
```

Comment:
List of project ids for which this category can be used. If project_ids is null, then this category can be used for any project